environment.yml

```
@@ -55,7 +55,7 @@ dependencies:
       # doc generation dependency
       - pygithub
       # mdanalysis and its tests and test data
-      - git+https://github.com/MDAnalysis/mdanalysis@develop#egg=mdanalysis&subdirectory=package
-      - git+https://github.com/MDAnalysis/mdanalysis@develop#egg=MDAnalysisTests&subdirectory=testsuite
+      - git+https://github.com/MDAnalysis/mdanalysis@fix-discovery#egg=mdanalysis&subdirectory=package
+      - git+https://github.com/MDAnalysis/mdanalysis@fix-discovery#egg=MDAnalysisTests&subdirectory=testsuite
```

```suggestion
      - git+https://github.com/MDAnalysis/mdanalysis@develop#egg=mdanalysis&subdirectory=package
      - git+https://github.com/MDAnalysis/mdanalysis@develop#egg=MDAnalysisTests&subdirectory=testsuite
```
To revert back to `develop` once https://github.com/MDAnalysis/mdanalysis/pull/4508 is merged